geometry_msgs/PointStamped pickupPos
geometry_msgs/PointStamped placePos
string job
bool drop 
